include/uv-win.h

```diff
@@ -647,3 +647,17 @@ RB_HEAD(uv_timer_tree_s, uv_timer_s);
 #ifndef X_OK
 #define X_OK 1
 #endif
+
+#if _MSC_VER >= 1900
+#undef stdin
```

Comment:
OOP, my mistake! 
This seems to be bug. 
I rechecked the case, here is compile error output:
i have created another simple console application,  compile error again,
link above libs does not help.

#include <stdio.h>
#include <stdlib.h>
#include "uv.h"

#pragma comment(lib, "libuv.lib")

#pragma comment(lib, "advapi32.lib")
#pragma comment(lib, "iphlpapi.lib")
#pragma comment(lib, "psapi.lib")
#pragma comment(lib, "shell32.lib")
#pragma comment(lib, "user32.lib")
#pragma comment(lib, "userenv.lib")
#pragma comment(lib,"ws2_32.lib")


int main()
{
...




1>------ Build started: Project: ConsoleApplication1, Configuration: Debug Win32 ------
1>  stdafx.cpp
1>  ConsoleApplication1.cpp
1>LINK : warning LNK4098: defaultlib 'LIBCMTD' conflicts with use of other libs; use /NODEFAULTLIB:library
1>libuv.lib(uv-common.obj) : error LNK2019: unresolved external symbol _fprintf referenced in function _uv__print_handles
1>libuv.lib(error.obj) : error LNK2001: unresolved external symbol _fprintf
1>libuv.lib(error.obj) : error LNK2019: unresolved external symbol ___iob_func referenced in function _uv_fatal_error
1>libuv.lib(util.obj) : error LNK2019: unresolved external symbol __snwprintf referenced in function _uv_cpu_info
1>libuv.lib(fs-event.obj) : error LNK2001: unresolved external symbol __snwprintf
1>libuv.lib(fs.obj) : error LNK2001: unresolved external symbol __fmode
1>libuv.lib(snprintf.obj) : error LNK2019: unresolved external symbol _vsnprintf_s referenced in function _snprintf
1>MSVCRTD.lib(vsnprintf_s.obj) : error LNK2001: unresolved external symbol _vsnprintf_s
1>libuv.lib(snprintf.obj) : error LNK2019: unresolved external symbol __vscprintf referenced in function _snprintf
1>libuv.lib(inet.obj) : error LNK2019: unresolved external symbol _sprintf referenced in function _inet_ntop6
1>MSVCRTD.lib(vsnprintf_s.obj) : error LNK2001: unresolved external symbol __vsnprintf_s
1>C:\ConsoleApplication1\Debug\ConsoleApplication1.exe : fatal error LNK1120: 8 unresolved externals
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========